components/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,33 @@
+import { AreWeHeadlessYetLogo } from '../../SVG';
+import styles from './Footer.module.scss';
+import type FooterProps from './types';
+
+function getYear(date: string) {
+    const dateObject = new Date(date);
+    return dateObject.getFullYear();
+}
+
+export const Footer = ({ lastPublishedAt }: FooterProps) => (
+    <footer className={styles.footer}>
+        <div className={styles.footer__text}>
+            <div>
+                <strong className={styles.copyright}>
+                    © Are we headless yet?
+                </strong>
```

Comment:
I see that this is supposed to render as not bold on mobile, but the `<strong>` tag adds font-weight bold by default - maybe replace this with a span?

Reply:
See [this comment from Thibaud](https://github.com/wagtail/areweheadlessyet/pull/1#discussion_r797520016) suggesting that 
> it's a design inconsistency for this kind of text to have different styles across viewports.

So I think we want to render it bold on mobile as well.